packages/vscode-extension/src/utilities/packageManager.ts

```diff
@@ -37,25 +70,39 @@ export async function resolvePackageManager(): Promise<PackageManagerInfo | unde
       } as const)
     );
 
-    const files = await fs.readdir(workspace);
+    const getDirFilesSortedByModificationDate = async (dir: string) => {
+      const files = await fs.promises.readdir(dir);
+
+      return files
+        .map((fileName) => ({
+          name: fileName,
+          time: fs.statSync(`${dir}/${fileName}`).mtime.getTime(),
+        }))
+        .sort((a, b) => a.time - b.time)
+        .map((file) => file.name);
+    };
+
+    const files = await getDirFilesSortedByModificationDate(workspace);
+    const packageManagerCandidates = [];
     for (const file of files) {
-      const packageManager = lockFiles.get(file);
-      if (packageManager) {
-        return packageManager;
+      const manager = lockFiles.get(file);
```

Comment:
do I see it wrong of lockFiles is a map with single entry "yarn.lock" ? does it even make sense to use it this way in this case?

Reply:
github cropped the rest: 
```
const lockFiles = new Map(
      Object.entries({
        "yarn.lock": "yarn",
        "package-lock.json": "npm",
        "pnpm-lock.yaml": "pnpm",
        "bun.lockb": "bun",
      } as const)
    );
```